cmd/upgrade/status.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+import (
+	"github.com/openebs/openebsctl/pkg/upgrade/status"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdUpgradeStatus() *cobra.Command {
```

Comment:
Can this command be removed in favour of `--wait` flag in the upgrade sub-command itself?
This will remove a requirement for another command, and will sort of match the recommended way suggested by [cobra CLI's UX and a lot of other *nix tool UX](https://github.com/spf13/cobra#concepts).

Reply:
Are you asking to remove the command completely or just the --wait flag for the command and shift it to the job creation time?

Reply:
There are two portions being one that shows logs and other that waits for the continuous logs.